README.md

```diff
@@ -30,12 +30,42 @@ Currently, these distributions have existing packages for HyFetch:
 * Nix: `nix-env -i hyfetch` (Thanks to @ YisuiDenghua)
 * Guix: `guix install hyfetch` (Thanks to @ WammKD)
 
+### Method 3: Install using git and python
+
+Run the following commands:
+```sh
+git clone https://github.com/hykilpikonna/hyfetch.git
+cd hyfetch
+sudo python setup.py install
+```
+
 ## Usage
 
 When you run `hyfetch` for the first time, it will prompt you to choose a color system and a preset. Just follow the prompt, and everything should work (hopefully). If something doesn't work, feel free to submit an issue!
 
 If you want to use the updated `neofetch` without LGBTQ flags, check out [this section](https://github.com/hykilpikonna/hyfetch#running-updated-original-neofetch)
 
+## Contributing
+
+### Building Hyfetch
+
+clone the git repository using:
+`git clone https://github.com/hykilpikonna/hyfetch.git`
+
+cd into the repository's directory
+
+cd into the "tools" directory
+
+run the following command:
+`./buildAndClean.sh`
+
+### Running the build
+
+go into the root of the hyfetch directory and run the following command:
+`python runner.py`
```

Comment:
Actually I wrote `runner.py` before I knew that modules can be executed with `python -m {module}`. So now you can run hyfetch from source by running `python -m hyfetch` from project root without building or using `runner.py`.

I can delete `runner.py` afterwards